src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator-container">
+        <input type="text"></input>
```

Comment:
- [ ] Your calculator design looks good. Just missing one thing, you have missed putting `0` on the display screen by default. Please make sure to update it.

### Your design
![Screenshot from 2021-09-23 02-43-52](https://user-images.githubusercontent.com/22434660/134423053-65568451-d49b-4b73-8bef-24d8f0312502.png)

### Given design

![Screenshot from 2021-09-23 02-46-49](https://user-images.githubusercontent.com/22434660/134423173-05618b1b-c9bf-49b2-bf9d-83ae82aa5a76.png)